src/Hostnet/Component/CodeSniffer/Installer.php

```diff
@@ -24,41 +26,35 @@ class Installer implements PluginInterface, EventSubscriberInterface
     private $io;
 
     /**
-     * Ugly Composer static call to hook in to the post-autoload-dump event.
-     *
-     * @return array holding the post-autoload-dump hook and function to execute.
+     * {@inheritdoc}
      */
-    public static function getSubscribedEvents()
+    public static function getSubscribedEvents(): array
     {
-        return ['post-autoload-dump' => 'execute'];
+        return [ScriptEvents::POST_AUTOLOAD_DUMP => 'execute'];
     }
 
-    /**
-     * Configuration for standalone use in a system wide installation scenario
```

Comment:
I think this is a useful comment that denotes something that is hard to know when reading the code - that this is about using the library stand alone. It's not a good thing to remove things like that from the source code.